roadmap/implementors-guide/src/types/overseer-protocol.md

```diff
@@ -172,11 +172,10 @@ If this subsystem chooses to second a parachain block, it dispatches a `Candidat
 
 ```rust
 enum PoVDistributionMessage {
-	/// Note a statement by a validator on a relay-parent. `Seconded` statements must always
-	/// have been passed in before `Valid` or `Invalid` statements.
-	ValidatorStatement(Hash, SignedFullStatement),
 	/// Fetch a PoV from the network.
-	/// (relay_parent, PoV-hash, Response channel).
+	///
+	/// This `CandidateDescriptor` should correspond to a candidate seconded under the provided
+	/// relay-parent hash.
 	FetchPoV(Hash, CandidateDescriptor, ResponseChannel<PoV>),
```

Comment:
```suggestion
	FetchPoV(Hash, CandidateDescriptor, oneshot::Sender<PoV>),
```

Reply:
In the guide we have used `ResponseChannel` instead of `oneshot::Sender` in general. It encapsulates the concept of sending a response over a channel without tying us to the `oneshot` concept which is an implementation detail.

Reply:
OK. I'm fine abstracting higher than the specific implementation, but I do think it's a good idea to distinguish in the guide between channels which will carry a single value, and those which will carry a stream of them over time. I don't think that `ResponseChannel` makes that distinction.